CONTRIBUTING.md

```diff
@@ -1,35 +1,41 @@
 # Contributing to this project
 
-Please take a moment to review this document in order to make the contribution
-process easy and effective for everyone involved.
-
-Following these guidelines helps to communicate that you respect the time of
-the developers managing and developing this open source project. In return,
-they should reciprocate that respect in addressing your issue or assessing
-patches and features.
-
+Thank you for being interested in contributing to `igraph`! We need the help of
+volunteers to keep the package going, so every help is welcome. You can help out
+the project in several different ways. This repository only hosts the C code of
+the `igraph` project. Even if you are not so experienced with C, you can
+contribute in a number of ways.
+
+You can contribute to `igraph` in various ways:
+
+1. Respond to user questions on
+   our [support forum](https://igraph.discourse.group/).
+2. Correct or improve our [documentation](https://igraph.org/c/html/latest/).
+3. Go over [open issues](https://github.com/igraph/igraph/issues):
+   - Are some older issues still a problem in the most recent version?
+   - Can you reproduce some of the bugs that are reported?
+   - Some [issues point out problem with documentation](https://github.com/igraph/igraph/labels/documentation), perhaps you could help correct this?
+   - Looking to contribute code? Take a look at
+     some [good first issues](https://github.com/igraph/igraph/labels/good%20first%20issue).
 
 ## Using the issue tracker
 
-The issue tracker is the preferred channel for [bug reports](#bugs),
+- The issue tracker is the preferred channel for [bug reports](#bugs),
 [features requests](#features) and [submitting pull
-requests](#pull-requests), but please respect the following restrictions:
+requests](#pull-requests).
 
-* Please **do not** use the issue tracker for personal support requests (use
-  our [igraph support forum](https://igraph.discourse.group)).
-
-* Please **do not** derail or troll issues. Keep the discussion on topic and
-  respect the opinions of others.
-
-Please also take a look at our [tips on writing igraph code](#tips) before
-getting your hands dirty.
+- Do you have a support question? Please use
```

Comment:
"Do you have questions?"

Any _question_ should go to the forum, no need to say "support" question.